index.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Text Encrypter & Decrypter</title>
+</head>
+<body>
+    <h1>Text Encrypter & Decrypter</h1>
+    <label for="inputText">Enter Text:</label>
```

Comment:
I would suggest a line break after this, e.g. a `<br>` or wrapping the tags in `<p>` tags.